src/LaunchDarkly.EventSource/Configuration.cs

```diff
@@ -7,18 +7,19 @@
 namespace LaunchDarkly.EventSource
 {
     /// <summary>
-    /// A class used 
+    /// An immutable class containing configuration properties for <see cref="EventSource"/>.
```

Comment:
Unrelated to the feature PR, and a bit embarrassing: this doc comment has apparently said nothing but "A class used" for the last 3 years.